tests/tc_ufcs_struct_completion/fooutils/fooutils.d

```diff
@@ -13,4 +18,6 @@ void ufcsBarRefConstWrapped(ref const(Foo) foo, string mama) {}
 void ufcsBarRefImmuttableWrapped(ref immutable(Foo) foo, string mama) {}
 void ufcsBarScope(ref scope Foo foo, string mama) {}
 void ufcsBarReturnScope(return scope Foo foo, string mama) {}
+void ufcsSomeInt(int x) {}
+void ufcsSomeShort(short x) {}
```

Comment:
also add a line

`void ufcsExactAliasedType(IntAliased x) {}`

to confirm that the real type itself still auto-completes as well.